taxonomies/food/ingredients.txt

```diff
@@ -3919,8 +3109,9 @@ ciqual_food_code:en: 19250
 ciqual_food_name:en: Sheep milk, whole
 ciqual_food_name:fr: Lait de brebis, entier
 
+# < en:skimmed milk
+< en:milk
 < en:sheeps milk
-< en:skimmed milk
 en: skimmed sheeps milk, skimmed sheep milk, semi-skimmed sheeps milk
 ar: حليب الغنم قليل الدسم
 bg: частично обезмаслено овче мляко
```

Comment:
Is there a reason this section is kept?

Reply:
If you want a list of skimmed (goat, cow, ..) milks?

Reply:
But why keep skimmed goat milk, but (all but) remove skimmed (cow) milks? I don’t understand the logic of moving processing out of ingredients, but then keeping some processing in ingredients.

Reply:
It is should not have been removed if I follow the nutritional value logic.